src/ShallowTestWrapper.js

```diff
@@ -2,6 +2,22 @@ import $ from 'cheerio'
 
 import TestWrapper from './TestWrapper'
 
+function getRoot (wrapper) {
+  // doing this to maintain backwards compatibility with enzyme 2.x
+  if (typeof wrapper.root === 'function') {
+    return wrapper.root()
+  }
+  return wrapper.root
+}
+
+function getRootName (wrapper) {
+  const root = getRoot(wrapper)
+  const instance = root.instance()
+  const fn = instance && instance.constructor
+  const name = fn ? fn.displayName || fn.name : root.name();
```

Comment:
sadly i think this line needs to have its semicolon removed

Reply:
@ljharb when will the PR be merged so we can get this resolved?

Reply:
When tests pass.